client/galaxy/scripts/mvc/collection/list-of-pairs-collection-creator.js

```diff
@@ -568,7 +568,8 @@ var PairedCollectionCreator = Backbone.View.extend( baseMVC.LoggableMixin ).exte
      */
     createList : function( name ){
         var creator = this,
-            url = '/api/histories/' + this.historyId + '/contents/dataset_collections';
+            root = ( window.Galaxy && Galaxy.options.root )? Galaxy.options.root : '/',
```

Comment:
> Where would that function live as far as scope? If you make it window, you'd still need the ternary operator (outside the function):
root = window.get_galaxy_root? get_galaxy_root() : '/',
Is that what your were hoping to clear up or something else?

Yes, this one.

> The Galaxy object should be available on every page at this point, but it doesn't hurt to do the fall back as a bit of defensive programming.

Ok, makes sense. Thanks for the explanation.